website/src/main/tut/docs/rules/DisableSyntax.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+title: Disable
+---
+
+# DisableSyntax
+
+_Since 0.5.4_
+
+This rule reports errors when a "disallowed" syntax is used.
```

Comment:
Explain what we mean by syntax? Essentially, just make it clear this does not require types or compilation.